src/components/common/typeahead-dropdown/typeahead-dropdown.ts

```diff
@@ -18,6 +20,9 @@ export class TypeaheadDropdownComponent {
   @Input()
   property: string;
 
+  @Output()
+  candidateChange = new EventEmitter<string>();
```

Comment:
think the use of <string> here is incorrect, would assume we are emitting some sort of object i.e. 
{
  name: string,
  slot: TestSlot,
}